src/cljs/replumb/repl.cljs

```diff
@@ -759,11 +779,11 @@
   ([]
    (reset-env! nil))
   ([namespaces]
+   (read-eval-call {} identity "(in-ns 'cljs.user)")
```

Comment:
The links are broken (I have been bitten too) when you change the file...but I think I have understood.